course/src/isometry.cpp

```diff
@@ -10,9 +10,169 @@
 namespace ekumen {
 namespace math {
 
-/*
- * your code here
- */
+namespace {
+
+Matrix3 matrix_multiplication(const Matrix3& matrix1, const Matrix3& matrix2) {
+  Matrix3 aux;
+  for (int i = 0; i < 3; ++i) {
+    for (int j = 0; j < 3; ++j) {
+      for (int iter = 0; iter < 3; ++iter) {
+        aux[i][j] += matrix1[iter][i] * matrix2[j][iter];
+      }
+    }
+  }
+  return aux;
+}
+
+Vector3 translation_vector_operation(const Matrix3& matrix,
+                                     const Vector3& first_vector,
+                                     const Vector3& second_vector) {
+  Vector3 aux;
+  aux = matrix * first_vector + second_vector;
+  return aux;
+}
+
+Matrix3 transpose(const Matrix3& matrix) {
```

Comment:
Same thing here.
Transposing is a typical thing related to matrixes. So I would move this function to `Matrix3`  class.
```c++
Matrix3 transpose();
```